modules/fieldlinessequence/rendering/renderablefieldlinessequence.h

```diff
@@ -93,6 +107,8 @@ class RenderableFieldlinesSequence : public Renderable {
     int _activeStateIndex = -1;
     // Active index of _startTimes
     int _activeTriggerTimeIndex = -1;
+    // Manual time offset
+    double _manualTimeOffset = 0.0;
     // Number of states in the sequence
     size_t _nStates = 0;
     // In setup it is used to scale JSON coordinates. During runtime it is used to scale
```

Comment:
The `_sequenceEndTime` variable is not initialized to a proper value so the first comparison might behave very strangely as the variable will contain garbage